<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackendWidget</class>
 <widget class="QWidget" name="BackendWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listView">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDropMode::InternalMove</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectionBehavior::SelectItems</enum>
     </property>
     <property name="movement">
      <enum>QListView::Movement::Free</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="addButton">
       <property name="toolTip">
        <string>Add a new Place</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset theme="list-add"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editButton">
       <property name="toolTip">
        <string>Edit currently selected Place</string>
       </property>
       <property name="text">
        <string>Edit selected</string>
       </property>
       <property name="icon">
        <iconset theme="gtk-edit"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="removeButton">
       <property name="toolTip">
        <string>Remove currently selected Place</string>
       </property>
       <property name="text">
        <string>Remove selected</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="syncButton">
       <property name="toolTip">
        <string>Sync from this backend</string>
       </property>
       <property name="text">
        <string>Sync from this backend</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
